.github/workflows/ci.yaml

```
@@ -0,0 +1,108 @@
+name: CI Pipeline
+
+on:
+  workflow_call:
+    secrets:
+      MAIL_USERNAME:
+        description: 'Username for the email notification step'
+        required: true
+      MAIL_PASSWORD:
+        description: 'Password for the email notification step'
+        required: true
+    inputs:
+      custom_docker_tag:
+        description: 'Custom Docker tag'
+        # If not specified, then the 'latest' tag will be used
+        default: ''
```

then your default value should be set to latest
```suggestion
        # If not specified, then the 'latest' tag will be used
        default: latest
```

---

Fixed